src/SPECS/hootenanny.spec.in

```diff
@@ -187,19 +194,18 @@ if [ "$1" = "1" ]; then
         RAND_PW=$(pwgen -s 16 1)
         sudo -u postgres createuser --superuser hoot || true
         sudo -u postgres psql -c "alter user hoot with password '$RAND_PW';"
-        sudo sed -i s/DB_PASSWORD=.*/DB_PASSWORD=$RAND_PW/ /var/lib/hootenanny/conf/DatabaseConfig.sh
+        if [ -f /var/lib/hootenanny/conf/DatabaseConfigDefault.sh ]; then
+            sudo echo "export DB_PASSWORD=$RAND_PW" > /var/lib/hootenanny/conf/DatabaseConfigLocal.sh
+        else
+            sudo sed -i s/DB_PASSWORD=.*/DB_PASSWORD=$RAND_PW/ /var/lib/hootenanny/conf/DatabaseConfig.sh
+        fi
         sudo -u postgres createdb hoot --owner=hoot
         sudo -u postgres createdb wfsstoredb --owner=hoot
         sudo -u postgres psql -d hoot -c 'create extension hstore;'
         sudo -u postgres psql -d postgres -c "UPDATE pg_database SET datistemplate='true' WHERE datname='wfsstoredb'"
         sudo -u postgres psql -d wfsstoredb -c 'create extension postgis;'
     fi
 
-    # restore saved db config file if present
-    if [ -f /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave ]; then
-        sudo mv /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave /var/lib/hootenanny/conf/DatabaseConfig.sh
-    fi
-
```

Comment:
I think we should restore this check for an rpmsave file and replace the mv command with something like:
`sudo grep DB_PASSWORD /var/lib/hootenanny/conf/DatabaseConfigDefault.sh > /var/lib/hootenanny/conf/DatabaseConfigLocal.sh`
so that a package update will get the local config file created.